core/spec/models/spree/line_item_spec.rb

```diff
@@ -110,6 +110,16 @@
     end
   end
 
+  # test for copying prices when the vat changes
+  context '#update_price' do
+    it 'copies over a variants differing price for another vat zone' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.